laravel/the-basics/databases.html.md

```diff
@@ -180,6 +180,178 @@ fly secrets set DB_USERNAME=<DB_USERNAME> DB_PASSWORD=<DB_PASSWORD>
 fly deploy 
 ```
 
+## _Postgres in Fly.io_
+We have a <i>[whole section](/docs/postgres/)</i> dedicated for [Postgres](https://www.postgresql.org/), we even have a `flyctl` integration to simplify the creation and management of your Postgres Fly Apps!
+
+In this section you'll get to quickly:
+1. Setup a Postgres Fly App
+2. Connect From a Laravel Fly App by Attachment
+3. Connect From a Laravel Fly App Manually
+4. Connect From a Local Environment
+5. Test Connection 
+
+### _Setup a Postgres Fly App_
+1) You can easily setup your Postgres in Fly.io by utilizing our `flyctl postgres create` command:
+
+```cmd
+fly postgres create
+```
+
+<small>Running the command should give you several prompts, one of which is a prompt for your resource allocation. Be sure to check out the appropriate [configuration plan](/docs/reference/postgres/#about-free-fly-postgres) based on your usecase.</small>
+
+
+2) Soon after you should receive the auto-generated configuration for your Redis Fly App. Make sure to <b>keep them somewhere safe</b>, there will be no place to find them afterwards!
+```output
+Postgres cluster ktan-pg created
+  Username:    postgres
+  Password:    <redacted>
+  Hostname:    ktan-pg.internal
+  Proxy Port:  5432
+  Postgres Port: 5433
+```
+
+### _Interacting with your Postgres Fly App_
+Now that you have your Postgres Fly App running, how do you interact with it? Notice how there wasn't any `fly.toml` configuration file generated during the whole setup process above.
+
+Fear not! You can interact with your Postgres Fly App using `flyctl postgres connect`. 
+
+1) Connect with your Postgres Fly App using its App name. 
+
+```cmd
+flyctl postgres connect -a ktan-pg
+```
+```output
+Connecting to ktan-pg.internal... complete
+psql (14.4 (Debian 14.4-1.pgdg110+1))
+Type "help" for help.
+
+postgres=# 
+```
+<small>In case you've forgotten about your app name, but managed to save the output from the Postgres Fly App creation (which you should've!), then check the value of the `Hostname`. The App name would simply be every little character before ".internal".</small> 
+
+
+2) Great, you're connected! Go ahead and complete your setup with a new database:
+```postgres
+create database <DB_NAME>;
+```
+
+3) Now if you check your database list, you should find the recent database created:
```

Comment:
I don't if you this but there is `fly postgres db list` command that runs this query for you.

Reply:
*--super-excited typing--->*

Hi @rugwirobaker! It's awesome to have you in the Laravel Docs section! 

Now, for the step 3 above, it comes from a walkthrough of connecting with the Postgres Fly App through the `fly postgres connect -a <appname>" command. My idea was to show the reader the option of how they can connect to the Postgres Fly App from their console _then_  create a db from there, and of course while they are at it, why not list the db they've created from the console.

But it's great you've pointed out that there is a separate command for listing the Postgres Fly App's list of db using `fly postgres db list -a <appname>`, it's a great addition to the list of ways/approaches to interact with the postgres fly app, I'll add that command in there  in my next commit, and I'll be sure to add you in as a reviewer!